### ISSUE TITLE
Error: Too few positional arguments: 5 required, 4 given.

### ISSUE THREAD

**Issue by cedx** [cedx/grinder-coveralls#2]
When I try to collect the code coverage, this error is raised:

```
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/grinder_coveralls-1.1.0/lib/src/collector.dart:119:29: Error: Too few positional arguments: 5 required, 4 given.
      report = await collect(
                            ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/coverage-0.13.2/lib/src/collect.dart:28:30: Context: Found this candidate, but the arguments don't match.
Future<Map<String, dynamic>> collect(Uri serviceUri, bool resume,
```

**Comment by cedx** [comment]
Too bad: a breaking change was introduced by the `coverage` package in version 0.13.2, but the versioning did not respect SemVer (i.e. new version should be 0.14.0 instead of 0.13.2).

**Comment by cedx** [comment]
Fixed in version 1.1.1 👍
